src/RealtimeEffectPanel.cpp

```diff
@@ -495,8 +495,17 @@ namespace
                mEffectState->SetActive(mEnableButton->IsDown());
                if (mProject)
                {
-                  ProjectHistory::Get(*mProject).ModifyState(false);
-                  UndoManager::Get(*mProject).MarkUnsaved();
+                  auto const effectName{GetEffectName(*mEffectState)};
+                  ProjectHistory::Get(*mProject).PushState(
+                     /*! i18n-hint: undo history record
+                     first parameter - realtime effect name
+                     */
+                     XO("Change settings for effect %s").Format(effectName),
```

Comment:
Should we mention "realtime effect" instead of just "effect", to be consistent with
![image](https://github.com/user-attachments/assets/4337b34b-bbe4-4bbc-a25a-68a02afc3967) ?